simulation-runner/src/main/scala/com/puppetlabs/gatling/node_simulations/PerfTestLarge.scala

```diff
@@ -0,0 +1,72 @@
+package com.puppetlabs.gatling.node_simulations
+import com.puppetlabs.gatling.runner.SimulationWithScenario
+import org.joda.time.LocalDateTime
+import org.joda.time.format.ISODateTimeFormat
+import java.util.UUID
+
+import scala.concurrent.duration._
+
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+
+class PerfTestLarge extends SimulationWithScenario {
+
+	val reportBody = ElFileBody("PerfTestLarge_0006_request.txt")
+
+	val baseHeaders = Map("Accept" -> "application/json, text/pson",
+		"Accept-Encoding" -> "gzip;q=1.0,deflate;q=0.6,identity;q=0.3",
+		"User-Agent" -> "Puppet/5.3.3 Ruby/2.4.2-p198 (x86_64-linux)")
+
+	val headers_0 = baseHeaders ++ Map("X-Puppet-Version" -> "5.3.3")
```

Comment:
looks like this could be merged into baseHeaders.  but i realize this was copied in here, as-is.